### PR TITLE
Ienc event submission snugging v1

### PR DESCRIPTION
**Pull request by Nicklas-Coker**
This PR has the following check list against it. Due to the fields being defined in Gigexpert you may have to review this PR ( https://github.com/evvnt/gig-expert-api/compare/ienc-event-submission-snagging-v1?expand=1 )with this as well.
75 character Event Title validation - on screen input counter & validation
Remove 'Clear' and 'Done' date calendar picker buttons
Event Prices - add another price shouldn’t prefill any data
Next / Previous button icon not loading
Event Links - add another link shouldn't profile any data

### CODE REVIEW

File: dist/app/js/evvnt-submission-form-angular.js [hunk 1]
```diff
@@ -1938,6 +1940,9 @@ angular.module('evvnt.evvntSubmissionForm')
                 newsection = angular.copy(lastSection);
             }
             repeatsection.push(newsection);
+            // Clear the last copied fields
```
Comment (andypike) [inline dist/app/js/evvnt-submission-form-angular.js]: You might want to consider removing comments like this as they are really just noise.

File: dist/app/js/evvnt-submission-form-angular.js [hunk 2]
```diff
@@ -2031,6 +2035,11 @@ angular.module('evvnt.evvntSubmissionForm')
                 newsection = angular.copy(lastSection);
             }
             repeatsection.push(newsection);
+            console.log(newsection);
+            // Remove last entries
+            newsection.currency    = null;
+            newsection.description = null;
+            newsection.price      = null;
```
Comment (andypike) [inline dist/app/js/evvnt-submission-form-angular.js]: If your style is to line up `=` then you should do this one too.

File: gulpfile.js [hunk 3]
```diff
@@ -91,14 +91,15 @@ gulp.task('bower', function() {
...
+var styles = ['./styles/*','./node_modules/leaflet/dist/leaflet.css','./bower_components/angular-wizard/dist/angular-wizard.css',
+    './bower_components/ui-select/dist/select.min.css', './bower_components/components-font-awesome/css/font-awesome.min.css']
 
 gulp.task('build-css', function() {
     return gulp.src(styles)
         .pipe(sourcemaps.init())
         .pipe(sass())
         //.pipe(cachebust.resources())
-        .pipe(sourcemaps.write('./maps'))
+       // .pipe(sourcemaps.write('./maps'))
```
Comment (andypike) [inline gulpfile.js]: Personally I'd remove commented out code if it isn't needed. If you need to refer back to this then you can get it from git history.

File: js/controllers/prices-controller.js [hunk 4]
```diff
@@ -48,6 +48,11 @@ angular.module('evvnt.evvntSubmissionForm')
                 newsection = angular.copy(lastSection);
             }
             repeatsection.push(newsection);
+            console.log(newsection);
```
Comment (andypike) [inline js/controllers/prices-controller.js]: You may want to remove `console.log` calls before merging.

Comment (Nicklas-Coker) [inline js/controllers/prices-controller.js]: Thanks. This has been removed.

File: gulpfile.js (hunk 3 above)
Comment (Nicklas-Coker) [inline gulpfile.js]: Thanks. This has been removed.

File: dist/app/js/evvnt-submission-form-angular.js (hunk 2 above)
Comment (Nicklas-Coker) [inline dist/app/js/evvnt-submission-form-angular.js]: Thanks. This has been aligned.

File: dist/app/js/evvnt-submission-form-angular.js (hunk 1 above)
Comment (Nicklas-Coker) [inline dist/app/js/evvnt-submission-form-angular.js]: Thanks. This has been removed.

**Comment by andypike** [comment]
:+1: :shipit: (Just fix those merge conflicts)

**Comment by Nicklas-Coker** [comment]
thanx. I'm on it :-)

**Pull request merged**